src/components/network/network-panel.js

```diff
@@ -137,6 +141,14 @@ genotet.NetworkPanel.prototype.setEdgeInfo_ = function(edge, container) {
     .text(edge.target.label);
   container.children('#weight').children('span')
     .text(edge.weight);
+  container.children('#rm-edge').children('button').click(function() {
+    this.signal('update', {
+      type: 'delete-edge',
+      source: edge.source.label,
```

Comment:
This edge comes from the edge notation in renderer, each of edge.source and edge.target contains the label for that gene. So it works correctly for the removal function.